frontend/web/components/pages/HomeAside.tsx

```diff
@@ -252,6 +252,14 @@ const HomeAside: FC<HomeAsideType> = ({
                                         <IonIcon className='mr-2' icon={code} />
                                         SDK Keys
                                       </NavLink>
+                                      <NavLink
+                                        id='identity-link'
+                                        exact
+                                        to={`/project/${project.id}/environment/${environment.api_key}/sdk-identities`}
+                                      >
+                                        <IonIcon className='mr-2' icon={statsChart} />
+                                        Identities Analytics
```

Comment:
It feels like this should sit as a sub page underneath identities, rather than having an "Identities" and "Identities Analytics" link

Reply:
Looking at the actual ui, the chart may as well sit above the identities table.